release/src/router/rom/webs_scripts/merlin_webs_update.sh

```diff
@@ -37,14 +37,13 @@ if [ "$?" != "0" ]; then
 	nvram set webs_state_error=1
 else
 
-	fullver=$(grep $model /tmp/wlan_update.txt | sed s/.*#FW//;)
+	fullver="$(grep "$model" /tmp/wlan_update.txt | tail -n1 | sed 's/.*#FW//')"
 	fullver=$(echo $fullver | sed s/#.*//;)
 	firmbase=$(echo $fullver | cut -d. -f1)
 	firmver=$(echo $fullver | cut -d. -f2)
 	buildno=$(echo $fullver | cut -d. -f3)
 
-	extendno=$(grep $model /tmp/wlan_update.txt | sed s/.*#EXT//;)
-	extendno=$(echo $extendno | sed s/#.*//;)
```

Comment:
The previously existing line **47** should remain as it was originally (fix only line **46**):
`extendno=$(echo $extendno | sed s/#.*//;)`

Again, because we want to maintain full backward compatibility & functionality, we change only what fixes the issue without disturbing anything else, especially since we don't have the hardware to test & verify whatever many variations exist in versioning strings for GNUton F/W updates.

Reply:
@Martinski4GitHub !

Added it back in line 47 as seen above
Strictly technically speaking I don't think it's required to get the output we had, but I do understand your concerns around maintaining backward combability and functionality so I've added it back in after the first extendno is populated.